AppConfig.js

```diff
@@ -24,27 +24,32 @@ var config = {
       kraken: {
         name: 'Kraken',
         orgUrl: 'https://www.kraken.com/',
-        url: 'https://api.kraken.com/0/public/Ticker?pair=DASHUSD'
+        url: 'https://api.kraken.com/0/public/Ticker?pair=DASHUSD',
+        urlEur: 'https://api.kraken.com/0/public/Ticker?pair=DASHEUR',
+        urlBTC: 'https://api.kraken.com/0/public/Ticker?pair=DASHXBT'
       },
       poloniex: {
         name: 'Poloniex',
         orgUrl: 'https://poloniex.com/',
-        url: 'https://poloniex.com/public?command=returnTicker'
+        url: 'http://dashpay.info/api/v0/prices'
```

Comment:
https://dashpay.info/api/v0/prices

Reply:
What is `dashpay.info`? Why are we depending on this external services vs using a Poloniex URL for Poloniex price data?

Reply:
@nmarley thats API owned by our mobile dev team. I'd like to use it because original API pretty often cause a captcha.